runner/app/routes/llm_generate.py

```diff
@@ -0,0 +1,123 @@
+import logging
+import os
+from typing import Annotated, Optional, List
+from fastapi import APIRouter, Depends, Form, status, Request
+from fastapi.responses import JSONResponse, StreamingResponse
+from fastapi.security import HTTPAuthorizationCredentials, HTTPBearer
+from app.dependencies import get_pipeline
+from app.pipelines.base import Pipeline
+from app.routes.util import HTTPError, LlmResponse, TextResponse, http_error
+import json
+
+router = APIRouter()
+
+logger = logging.getLogger(__name__)
+
+RESPONSES = {
+    status.HTTP_200_OK: {"model": LlmResponse},
```

Comment:
I believe the only thing missing is adding the response name override for the SDK like this 
https://github.com/livepeer/ai-worker/blob/5994f25b777f1dc7ee66ad5a1c33c29acfe0f116/runner/app/routes/image_to_video.py#L25